Source/Atomic/CMakeLists.txt

```diff
@@ -226,3 +235,18 @@ target_include_directories (Atomic PUBLIC
 )
 
 include(AtomicDoc)
+
+if ($ENV{ATOMIC_BUILD_DIST})
+    target_compile_definitions(Atomic PUBLIC -DATOMIC_BUILD_DIST=1)
+endif ()
+
+if (MSVC)
+    target_compile_options(Atomic PUBLIC "$<$<CONFIG:Debug>:${ATOMIC_MSVC_RUNTIME}d>")
```

Comment:
I was combing through and looking to see we didn't lose linking to debug runtime.  Smiled here: ```${ATOMIC_MSVC_RUNTIME}d```, great attention to detail! 🎸